eventhubs/data-plane/azure-eventhubs-eph/pom.xml

```diff
@@ -1,19 +1,23 @@
-<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                             http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>azure-eventhubs-eph</artifactId>
```

Comment:
Should probably include a groupId in here of `com.microsoft.azure`?

Reply:
eventhubs-parent has three modules, and all modules pom files don't has groupId included. If we decide to include in EPH, then i will updates same to other two pom files?

Reply:
updated to all EventHubs' pom files